MSAL/src/telemetry/MSALTelemetry.h

```diff
@@ -21,22 +21,7 @@
 // OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 // THE SOFTWARE.
 
-/*!
- @protocol MSALDispatcher
- 
- Developer should implement it in order to receive telemetry events.
- 
- Usage: an instance of MSALDispatcher implementation is required when registerring dispatcher for MSALTelemetry.
- */
-@protocol MSALDispatcher <NSObject>
-
-/*!
- Callback function that will be called by MSAL when telemetry events are flushed.
- @param events events is represented by an array of dictionary of key-value pair of event property name/value.
- */
-- (void)dispatchEvent:(nonnull NSArray<NSDictionary<NSString *, NSString *> *> *)events;
```

Comment:
Someone might be using those. Should we still keep those one public but deprecate and recommend usage of config. Otherwise it will break the app on upgrade. Same for logger.

Reply:
Let me move implementation to the config and call these deprecated.